doc/source/quickstart.rst

```diff
@@ -0,0 +1,92 @@
+===========
+Quick Start
+===========
+
+The quick start describes step by step how to install the testbed.
+
+The prerequisite is to have an account on one of the supported OpenStack cloud
+providers. It is not part of this quick start to describe the registration with
+the individual cloud providers. Please contact the respective cloud provider
+for this.
```

Comment:
Maybe add: Deploying on other OpenStack clouds should be possible, but you may need to adapt some settings, like flavor and image names.

Also adding a link to the quota that is needed would seem helpful.